trunk/includes/pardot-settings-class.php

```diff
@@ -860,69 +886,33 @@ function clearcache_field() {
 
 	/**
 	 * Encrypts with a bit more complexity
-	 *
+	 * returns false if the string could not be encrypted (cases where encryption fails, or Sodium or OpenSSL are not present in PHP).
```

Comment:
Since the return value of false is never handled by anything calling this, perhaps errors should instead throw an exception instead of silently failing and setting the stored password to false?